### PR TITLE
[Feature] Notification component page

### PR DESCRIPTION
**Pull request by LJKaski**
This PR adds the Notification component page to the site. It contains the relevant menu additions as well as locale entries. It also updates `suomifi-ui-components` dependency to 6.2.0, which contains the Notification component.

### CODE REVIEW

**Review by valger11 — DISMISSED** [review]

File: locale/fi/notification.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      ],
+      "links": [
+        {
+          "text": "",
+          "url": ""
+        }
+      ]
+    }
+  ],
+  "example.error.heading": "Et voi muokata tätä versiosta",
```
Comment (ketsappi) [inline locale/fi/notification.json:50]: > Et voi muokata tätä versio**s**ta

I suggest we remove the s-letter for this example to sound better.

File: locale/fi/notification.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    ""
+  ],
+  "sections": [
+    {
+      "title": "Mistä komponentti koostuu",
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Komponentti koostuu ilmoituksen tyyppiä kuvaavasta ikonista, otsikosta ja pääsisällöstä sekä \"sulje\" painikkeesta. Ilmoituksen pääsisällön alle on myös mahdollista lisätä toimintoelementtejä, kuten painikkeita tarpeen vaatiessa."
```
Comment (ketsappi) [inline locale/fi/notification.json:18]: > myös mahdollista lisätä toimintoelementtejä, kuten painikkeita, tarpeen vaatiessa.

I would add one comma (after the "**painikkeita**") for this to be easier for reader and maybe more grammatically correct (?)

File: locale/fi/notification.json (hunk 1 above)
Comment (LJKaski) [reply locale/fi/notification.json:50]: This should now be fixed.

File: locale/fi/notification.json (hunk 2 above)
Comment (LJKaski) [reply locale/fi/notification.json]: Removed the "tarpeen vaatiessa" from the end, since it was mostly redundant.

**Review by ketsappi — APPROVED** [review]

**Pull request merged**